README.md

```diff
@@ -137,6 +137,255 @@ To download file you should call **GetFile** method
 
 Full code you can see at [DownloadFileFromContactTest](https://github.com/sochix/TLSharp/blob/master/TLSharp.Tests/TLSharpTests.cs#L167)
 
+# Events Sample code
+```csharp
+  using System;
+  using System.Threading.Tasks;
+  using TeleSharp.TL;
+  using TLSharp.Core;
+  using System.Linq;
+  using TeleSharp.TL.Messages;
+  using System.Collections.Generic;
+
+  namespace TLSharpPOC
+  {
+      class MainClass
+      {
+          const int APIId = 0;
+          const string APIHash = "???";
+          const string phone = "???";
+          public static void Main(string[] args)
+          {
+              new MainClass().MainAsync(args).Wait();
+          }
+
+          private async Task MainAsync(string[] args)
+          {
+			  TelegramClient client = null;
+		  
+			  // -- if necessary, IP can be changed so the client can connect to the test network.
```

Comment:
this session stuff doesn't seem to be relevant to the updates code, so I would remove it (or move it somewhere else in the README)